dockerfiles/alpine_3.5_2.2

```diff
@@ -5,7 +5,11 @@ RUN addgroup -S tarantool \
     && adduser -S -G tarantool tarantool \
     && apk add --no-cache 'su-exec>=0.2'
 
-ENV TARANTOOL_VERSION=2.2.2-4-g4f8ac5999 \
```

Comment:
I think we can use 2.x dockerfile for all alpine images

Reply:
Current commit only makes structure refactoring, but the next updates will merge the Dockerfiles as much as possible.